### PR TITLE
CLDC-2548: Configure shield

### PR DESCRIPTION
**Pull request by SamSeed-Softwire**
Implement basic shield protections.

I've raised a ticket to implement additional shield protections in the future: https://dluhcdigital.atlassian.net/browse/CLDC-2857.

### CODE REVIEW

File: terraform/production/main.tf
Comment (SamSeed-Softwire) [inline terraform/production/main.tf:50]: Currently want this disabled everywhere until we get the green light.

File: terraform/modules/front_door/shield.tf
Comment (SamSeed-Softwire) [inline terraform/modules/front_door/shield.tf:1]: In the future if we add other shield protections not specific to our front door (e.g. the policy which allows the Shield Response Team to access our account) then we may want to move all the shield resources out into a separate module. I think it's fine here for now though.

Comment (SamSeed-Softwire) [inline terraform/modules/front_door/shield.tf:1]: In the future if we add other shield protections not specific to our front door (e.g. the policy which allows the Shield Response Team to access our account) then we may want to move all the shield resources out into a separate module. I think it's fine here for now though.

On the other hand if you think that having lots of count statements is messy, I'd understand if you wanted to move it out into a separate module now!

**Review by RachaelBooth — COMMENTED** [review]
LGTM once the checks are sorted

**Review by RachaelBooth — APPROVED** [review]

**Pull request merged**
